libs/jsonschema/extension.go

```diff
@@ -35,13 +35,20 @@ type Extension struct {
 	// compatible with the current CLI version.
 	Version *int `json:"version,omitempty"`
 
+	// Preview indicates launch stage (e.g. PREVIEW).
+	Preview string `json:"x-databricks-preview,omitempty"`
+
```

Comment:
Can you include where/how this is used? It is non-standard and ends up in the JSON schema.

I believe only for Python codegen.